docs/source/upgrade.rst

```diff
@@ -106,6 +111,17 @@ NCCL and cuDNN No Longer Included in Wheels
 NCCL and cuDNN shared libraires are no longer included in wheels (see `#4850 <https://github.com/cupy/cupy/issues/4850>`_ for discussions). 
 You can manually install them after installing wheel if you don't have a previous installation; see :doc:`install` for details.
 
+cuTENSOR Enabled in Wheels
+--------------------------
+
+cuTENSOR can now be used when installing CuPy via wheels.
+
+``cupy.cuda.{nccl,cudnn}`` Modules Needs Explicit Import
+--------------------------------------------------------
+
+Previously ``import cupy.cuda`` automatically made ``cupy.cuda.nccl`` and ``cupy.cuda.cudnn`` modules available.
```

Comment:
```suggestion
Previously ``cupy.cuda.nccl`` and ``cupy.cuda.cudnn`` modules were available when directly importing CuPy with ``import cupy.cuda``
```

Reply:
To be more precise, just doing `import cupy` made these modules available.
```suggestion
Previously ``cupy.cuda.nccl`` and ``cupy.cuda.cudnn`` modules were automatically imported.
```

Reply:
yeah, this sounds much better :)